02_Basic Operators.playground/section-1.swift

```diff
@@ -22,16 +22,18 @@ let (x, y) = (1, 2)
 let string = "hello, " + "world"
 
 //Nil Coalescing Operator
-//It's equivalent to (a ? b : c), but for optionals
+//Shorthand for (a != nil ? a! : b)
```

Comment:
This might be confusing though.
When doing `(a ? b : c)`, `b` is not necessarily `a!`. The idea is to refer to the generic ternary operator.